### DISCUSSION TITLE
2024/04/07/sf4-1work/

### DISCUSSION THREAD

**Discussion by giscus** [Guailoudou/pages#4] [Announcements]
# 2024/04/07/sf4-1work/

乖漏斗博客

https://blog.gldhn.top/2024/04/07/sf4-1work/

**Comment by Guailoudou** [comment]
`//`和`#`都是注释，代码为Python，都是ai写的，ai注释瞎用

**Reply by Guailoudou** [reply]
已经修改了，全部改为#

**Reply by Guailoudou** [reply]
让ai全部重写了

**Comment by Guailoudou** [comment]
AI写的，不保证准确率奥
